### ISSUE TITLE
startSeconds/endSeconds

### ISSUE THREAD

**Issue by goto-bus-stop** [troybetz/react-youtube#52]
Before v5, a video could be played from a specific time using `playerVars`:

```js
<YouTube
  url={url}
  opts={{ playerVars: { start: 15, end: 130 } }}
/>
```

But since v5 uses `{cue,load}VideoById`, this no longer works. The *VideoById functions can take an object with the videoId and startSeconds/endSeconds properties, though: https://developers.google.com/youtube/iframe_api_reference#loadVideoById

Proposed fix:

```js
<YouTube
  videoId={id}
  startSeconds={15}
  endSeconds={130}
/>
```

I'll be happy to PR this if it sounds good :)

**Comment by tjallingt** [comment]
```javascript
updateVideo() {
    // set player vars
    let hasPlayerVars = typeof this.props.opts.playerVars !== 'undefined';
    let startSeconds = hasPlayerVars ? this.props.opts.playerVars.start : null;
    let endSeconds = hasPlayerVars ? this.props.opts.playerVars.end : null;
    let autoplay = hasPlayerVars && this.props.opts.playerVars.autoplay === 1;
    // if autoplay is enabled loadVideoById
    if (autoplay) {
      this._internalPlayer.loadVideoById(this.props.videoId, startSeconds, endSeconds, 'default');
      return;
    }
    // default behaviour just cues the video
    this._internalPlayer.cueVideoById(this.props.videoId, startSeconds, endSeconds, 'default');
}
```
:warning: untested and un-lint-ed code.
I'd suggest something like this to restore the functionality instead of adding extra properties (unless there are reasons for that which I am missing...).

**Comment by goto-bus-stop** [comment]
Right, that's fine with me too :)

(My reason for the new prop approach is that I quite like separate props for most things, including `volume` for example, but it's probably out of scope for this issue.)

**Comment by tjallingt** [comment]
I agree props are a very clean way of handling component options but since we already support the player options through the `opts` prop I think it is more important to have that function as expected :smile:

**Comment by goto-bus-stop** [comment]
alright :) would you like to test & PR your snippet?

**Comment by tjallingt** [comment]
Ah I will in a second thanks :P

**Comment by goto-bus-stop** [comment]
lovely, thank you!

**Comment by troybetz** [comment]
Fixed by #53, thanks!
